app/controllers/EditionsController.scala

```diff
@@ -268,6 +268,19 @@ class EditionsController(db: EditionsDB,
     }
   }
 
+  def removeCollectionFromFront(frontId: String, collectionId: String) = EditEditionsAuthAction { req =>
```

Comment:
Question - do you think we should prevent the removal of a non-empty collection?